### ISSUE TITLE
includeCollections saves only the last collection

### ISSUE THREAD

**Issue by CristianoStav** [boly38/node-mongotools#39]
Hello, im trying to save the three collections as in the example, but aways saves only the last array item, im doing somethiong wrong ?

```
const options = new MTOptions({
   uri: DATABASE_URL,
   path: pathToBackup,
   db: process.env.DATABASE,
   includeCollections: [
      `plataformas`,
      `ideias`,
      `userorganizationprofiles`,
   ],
   port: process.env.MONGO_PORT,
   });
```

**Comment by boly38** [comment]
Hi @CristianoStav thanks for this report 👍 

linked to #38 

Hi, I looked at dump options quickly, and seems that include can be repeated like exclude.
I'll suggest a fix asap

**Comment by boly38** [comment]
So the fix PR I will suggest would be:
- a) mark `includeCollections` option as deprecated (code+doc)
- b) `includeCollections` option will continue to work like now and undertake the last string of the array **only**. 
- c) Using  `includeCollections` option will produce warn log to inform about deprecated state.
- d) add `collection` option (to map mongodump binary option) that only accept one string or an array with one string may be accepted too, but an array with length $gte 2 will produce an error.

@CristianoStav and @antonio-goncalves ; whate do you think about this suggest ? feel free to give your opinion right here.

**Comment by antonio-goncalves** [comment]
@boly38 and @CristianoStav Sorry for my late response.

When I suggested to include the option "includeCollection" and "excludeCollections" I didn't realized that was not possible to include several "--collection" options, so the bug bug described here is my bad.

@boly38 I agree with your suggestion

**Comment by boly38** [comment]
>I didn't realized that was not possible to include several "--collection" options, so the bug bug described here is my bad.
no worry this is a good feature to have, I was realy happy to include your PR, mistakes are part of software development, we go ahead and that's nice !

**Comment by boly38** [comment]
fixed in `1.3.0`
